index.js

```diff
@@ -1,1144 +0,0 @@
-
-peacock.dialogueOne = `\nWhat an absolutely dreadful night it has turned out to be. It was such a lovely evening up until the murder. Mr. Body, that poor soul. I never could quite get a read on him. He seemed to be pre-occupied looking around the house. I suspect he was an admirer of art. He seemed to be closely inspecting Mr. Green's artwork. Professor Plum may be able to give you more about the deceased. I saw the two of them speaking earlier in the evening. He is in the library.`;
-
-plum.dialogueOne = `\nAh, inspector! I've been meaning to speak to you. Quite horrible, the recent events, quite horrible indeed! I've been collecting my thoughts here and trying to regain some sense of composure. The books, they always calm me down! And well, as I've been turning the last few hours over in my head, it did occur to me that I have borne witness to something that may be of interest to you. You see, earlier in the evening I was relaxing in a nook of the library, my attention captured completely by a lovely collection of English Romantic poetry. I must have been decently obscured from my position in the nook, because at one point Mr. Green and Mr. Body passed through and did not notice that I was here! They were in the midst of a dispute, and though they spoke in hushed voices I could tell from their tones that something was wrong. I could not clearly make out much of what they were saying, but I believe the argument had something to do with art. Eventually the conversation came to a close and Mr. Green left the room, after which Mr. Body pull out a notebook from his back pocket and begin writing in it furiously. Oh, I'm so embarrassed for spying, but I worried that it would stir up trouble if I revealed my presence. I certainly did not want to involve myself in whatever they were arguing about, either.\nIn any event, I suspect that the notebook Mr. Body was writing in may hold some clues as to the nature of their conversation, so it may be worth your while to see if you can find it. That's all the information I have for you, though I wish I could be of greater help!`;
-
-green.dialogueTwo = `\nHello again, inspector. I hope the investigation is coming along. A real shame what has happened to our friend Mr. Body. It seems like only hours ago we were enjoying a game of snooker in the Billiard Room.`;
-
-white.dialogueOne = `\nInspector, please tell me you'll get to the bottom of all this. My anxiety is at an all-time high knowing we have a killer in our midst! I never did get to speak to Mr. Body at all tonight and it's a shame! He seemed like such an insightful fellow. Mr. Green was just telling me all about how interested Mr. Body was in his artwork. I've never much had an eye for art myself, but I surely have enjoyed walking around his Conservatory and looking at Mr. Green's wonderful plants! I highly suggest you check them out!\n\n*Mr. Green's eyes dart to Mrs. White nervously...*`;
-
```

Comment:
Similarly to the room descriptions, these could have been generated on the player objects

Reply:
We wanted a single section where the dialogue was assigned to the characters so that we could easily reference the flow of dialogue as we were building the story. We also didn't have an equal number of dialogue instances for each character so we thought it would be easier to assign dialogue to the character objects after the fact. Mr. Green has two dialogues while all the other characters only have one. If we added dialogue in through the character constructor we would have to add two dialogue arguments to the constructor, of which only Mr. Green has two. The rest of the characters would end up having an empty second dialogue assignment, no?

Reply:
Fixed